services/provider/server/server_test.go

```diff
@@ -350,7 +353,10 @@ func TestGetExternalResources(t *testing.T) {
 			data, err := json.Marshal(mockResoruce.Data)
 			assert.NoError(t, err)
 			assert.Equal(t, string(extResource.Data), string(data))
+
 		}
+		assert.True(t, reflect.DeepEqual(extResource.Labels, mockResoruce.Labels))
+		assert.True(t, reflect.DeepEqual(extResource.Annotations, mockResoruce.Annotations))
```

Comment:
We could use assert.Equal() it uses reflect.DeepEqual internally

Reply:
Both `labels` and `annotations` are of type map, which means that in order to use `assert.Equal()` I'd need to convert from map to []byte which is not straightforward. Therefore I've chosen this assertion instead so it's compact as a one-liner.